src/Storages/StorageMongoDB.h

```diff
@@ -49,7 +45,7 @@ class StorageMongoDB final : public shared_ptr_helper<StorageMongoDB>, public IS
     void connectIfNotConnected();
 
     const std::string host;
-    const uint16_t port; /// NOLINT
+    // const uint16_t port; /// NOLINT -- actually not used, needs checking
```

Comment:
same as host and options, only used to construct uri

Reply:
Done, tnx.